plugin.json

```
@@ -11,7 +11,7 @@
         }
     },
     "settings_schema": {
-        "header": "",
+        "header": "This plugin is used to send alert notifications from [Amazon AWS CloudWatch](https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS",
```

🎉 -- this seems like a much better solution overall. It also looks like could stand to document which of these fields support Markdown today -- I'll review the code and submit an update to the documentation.

---

Minor nit: should this end with a period?
```suggestion
        "header": "This plugin is used to send alert notifications from [Amazon AWS CloudWatch](https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS.",
```

---

Just a small comment. The plugin does not send any alerts it receives alert from sns. Maybe can update this part